README.md

```diff
@@ -167,6 +167,7 @@ Branch.getFirstReferringParams().then(function (res) {
 1. Branch Session
   + [setDebug](#setDebug)
   + [initSession](#initSession)
+  + [getMixpanelInstance](#getMixpanelInstance)
```

Comment:
Can you change this method name to `setMixpanelToken` and update the references below?